test/1.2/errors/T_WUNUSED_DML12.dml

```diff
@@ -8,6 +8,7 @@ device test;
 
 /// COMPILE-ONLY
 
+/// WARNING WEXPERIMENTAL dml12-compatibility.dml
```

Comment:
The method overrides in dml12-compatibility causes `WEXPERIMENTAL` spam. `int64_attr` already caused it due to its preexisting override of `get()`, but now all attribute templates causes it. I'm thinking we should tweak DMLC so that it doesn't emit `WEXPERIMENTAL` if it originates from dml12-compatibility.

Reply:
Did exactly that, after Erik agreed.